src/common/tvgMath.h

```diff
@@ -297,6 +297,15 @@ static inline T lerp(const T &start, const T &end, float t)
     return static_cast<T>(start + (end - start) * t);
 }
 
+template <>
```

Comment:
it doesn't work. template fun is called and bug is still visible

Reply:
hmm... did you test it? my side, it works.

Reply:
please check now. I forgot about declaration in .h.
but can you please tell why do we prefer such solution?

Reply:
@mgrudzinska inline comes with the pros/cons together. If you're sure inlining is better for a certain case, then do it. Otherwise please don't it. it's basic  in c/c++ programming. Not our preference.

This is my guidance for you.
- A. Can you suggest any obvious faster result (fps?) with it? 
yes: good to use.
no: go to question B.
- B. Can you suggest the binary size is same or less? 
yes: Why not? you can also use it.
no: Don't use it.